Formula/gq-cli.rb

```diff
@@ -0,0 +1,33 @@
+class GqCli < Formula
+  desc "Command-line interface for GQ"
+  homepage "https://goquant.io"
+  license "MIT"
+
+  bottle do
+    sha256 cellar: :any_skip_relocation, monterey: "b21e0a5f4238130218bfae10c6604a05263e9ef52f462af246d54a93f54821d1"
+  end
+
```

Comment:
Please don't add a `bottle` block in a new formula. This is not found in `homebrew/core`'s package registry anyway.